cpp/include/functions.h

```diff
@@ -15,6 +15,31 @@
  */
 #pragma once 
 
+/* ----------------------------------------------------------------------------*/
+
+/**
+ * @Synopsis Renumber source and destination indexes to be a dense numbering.
+ *
```

Comment:
Not sure if "dense renumbering" is standard. I'd recommend adding "The resulting renumbered output contains GDF_INT32 or GDF_INT64 indexes between 0 and the number of vertices minus 1".

Reply:
I left the word dense in, but added your words as well.  Actually put the GDF_INT32 comment in the definition of the output rather than in the synopsis.

Reply:
What about using "contiguous" instead of "dense"? If I am not mistaken, renumbering makes vertex ids to be contiguous, and I guess this is more specific than dense.